.github/workflows/kube-proxy.yml

```diff
@@ -0,0 +1,17 @@
+name: kube-proxy-images
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  build:
+    runs-on: windows-2019
+    steps:
+    - uses: actions/checkout@v2
+    - uses: azure/docker-login@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_SECRET }}
```

Comment:
did you manage set these in the repository?

Reply:
Ah crud I don't have permission on the kubernetes-sigs repo, looks like I'll need to change myself to an admin to do so